src/components/Layout/Header/NavBar/UserMenu.tsx

```diff
@@ -148,18 +150,13 @@ export const UserMenu: React.FC<{ onClick?: () => void }> = ({ onClick }) => {
   }
   return (
     <ButtonGroup width='full'>
-      <WalletButton
-        onConnect={handleConnect}
-        walletInfo={walletInfo}
-        isConnected={isConnected}
-        isDemoWallet={isDemoWallet}
-        isLoadingLocalWallet={state.isLoadingLocalWallet}
-      />
       <Menu>
-        <MenuButton
-          as={IconButton}
-          aria-label='Open wallet dropdown menu'
-          icon={<ChevronDownIcon />}
+        <WalletButton
+          onConnect={handleConnect}
+          walletInfo={walletInfo}
+          isConnected={isConnected}
+          isDemoWallet={isDemoWallet}
+          isLoadingLocalWallet={state.isLoadingLocalWallet}
```

Comment:
Might as well spread this guy on line 143, and then

```suggestion
          isLoadingLocalWallet={isLoadingLocalWallet}
```